tests/data/databases/locidex_dbs/locidex_salm/config.json

```diff
@@ -0,0 +1,12 @@
+{
+    "db_name": "Locidex database",
+    "db_version": "1.0.0",
+    "db_date": "2024-02-25",
+    "db_author": "tester",
+    "db_desc": "Testdatabase for salmonella in mikrokondo",
+    "db_num_seqs": 53,
+    "is_nucl": true,
+    "is_prot": true,
```

Comment:
I'm no microbiologist, but can something be both nucleotide and protein like this?

Reply:
This is locidex specific, it denotes the prescence of a database of nucleotides and proteins. The `is` is a bit misleading though, I agree.